Docker/base-images/entando-eap71-base/settings.xml

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Any way to share these settings files across the different projects? Not a huge issue since these won't change a lot but it would be good if we only defined this stuff once.